### PR TITLE
Bump dependencies + resolve some javadoc warnings

### PR DESCRIPTION
**Pull request by LesTR**

### CODE REVIEW

File: core/src/main/java/cz/o2/proxima/util/SerializableUtils.java [hunk 1]
```diff
@@ -39,7 +39,6 @@
       try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
           ObjectOutputStream oos = new ObjectOutputStream(baos)) {
         oos.writeObject(original);
-        oos.close();
```
Comment (je-ik) [inline core/src/main/java/cz/o2/proxima/util/SerializableUtils.java:42]: Are you sure? I think we must at least call `flush`.

**Review by je-ik — CHANGES_REQUESTED** [review]
I think we cannot simply not flush the ObjectOutputStream before reading the resulting byte array. Otherwise LGTM! Thanks. :+1:

File: core/src/main/java/cz/o2/proxima/util/SerializableUtils.java (hunk 1 above)
Comment (LesTR) [reply core/src/main/java/cz/o2/proxima/util/SerializableUtils.java:42]: Added flush

**Review by je-ik — APPROVED** [review]
:+1:

**Pull request merged**